code/game/gamemodes/umbrage/umbrage_abilities/devour_will.dm

```diff
@@ -0,0 +1,30 @@
+//After a brief charge-up, equips a temporary dark bead that can be used on a human to knock them out and drain their will, making them vulnerable to conversion.
+/datum/action/innate/umbrage/devour_will
+	name = "Devour Will"
+	id = "devour_will"
+	desc = "Creates a dark bead that can be used on a human to fully recharge psi and knock them out."
+	button_icon_state = "umbrage_devour_will"
+	check_flags = AB_CHECK_RESTRAINED|AB_CHECK_STUNNED|AB_CHECK_LYING|AB_CHECK_CONSCIOUS
+	psi_cost = 20
+	lucidity_cost = 0 //Baseline
+	blacklisted = 1
+	var/list/victims = list() //A list of people we've used the bead on recently; we can't drain them again so soon
```

Comment:
It used to appear in the profiler as " " or "()" (so no name or just proc parens), and when you spot something so weird taking a decent chunk of your cpu usage at world start, you investigate it.

these days it still appears, but as "Init()", basically the idea is:

```New()``` is ALWAYS called, ALWAYS, meaning you can put stuff in there instead of byond being like "Oh, I see you wanted this to be a list, I'll just give you an Init()"